### PR TITLE
[ADD] knowledge: main, editing and properties pages

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
Task ID: 3035887

### CODE REVIEW

File: content/applications/productivity/knowledge/articles_creation.rst [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+   * - :guilabel:`/Template`
+     - Add a template section that can be inserted in messages, terms & conditions, or description
+       in other applications.
+
+Properties
+==========
+
+Properties are fields containing data and that can be added to articles by any user with
+:guilabel:`can write` access. These fields are shared between all the child articles or article
+items that are under the same parent.
```
Comment (chiaraprattico) [inline content/applications/productivity/knowledge/articles_creation.rst:167]: Properties are fields containing data that can be added to articles by any user with :guilabel:`can write` access. These fields are shared between all the child articles or article items under the same parent.

File: content/applications/productivity/knowledge/articles_creation.rst [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+To hide the property sidebar panel, click the :guilabel:`gear (⚙)` button.
+
+Remove articles
+===============
+
+Articles can be deleted or archived.
+
+To delete an article, click the :menuselection:`vertical ellipsis button (⋮) --> Delete`. The
+article is moved to the :guilabel:`Trash` for 30 days before being permanently removed. To reinstate
```
Comment (chiaraprattico) [inline content/applications/productivity/knowledge/articles_creation.rst:238]: To reinstall it

Comment (chiaraprattico) [reply content/applications/productivity/knowledge/articles_creation.rst:238]: not sure if that's what you meant.

**Review by chiaraprattico — CHANGES_REQUESTED** [review]

File: content/applications/productivity/knowledge/articles_creation.rst (hunk 2 above)
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_creation.rst:237]: No no, I meant reinstate :) 
image

**Review by chiaraprattico — APPROVED** [review]

File: content/applications/productivity/knowledge/articles_creation.rst [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+.. image:: articles_creation/text-editor.png
+   :align: center
+
+To open the text editor, select the text you want to format, and the editor appears. Then, click on
+the buttons to change the text style (normal, header 1 --> 6, code, quote), the format (bold,
+italic, underlined, strikethrough, no format), the font and background colors, and the font size.
+You can also create lists and insert (or edit) links.
+
+To format a whole paragraph, type `/` anywhere in the text. Doing so opens the
+:doc:`powerbox <../../../developer/reference/frontend/odoo_editor>`, which allows:
```
Comment (jcs-odoo) [inline content/applications/productivity/knowledge/articles_creation.rst:70]: You can use an absolute path, here, since it goes back to the root of the content, anyway.
```suggestion
:doc:`powerbox </developer/reference/frontend/odoo_editor>`, which allows:
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
This review is incomplete, as I'd rather discuss it IRL, mostly to change the structure of the page and other future pages.

cc @chiaraprattico

File: content/applications/productivity/knowledge/articles_creation.rst [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+items that are under the same parent.
+
+.. note::
+   To be able to add properties, an article must be either a **child article** or an **article
+   item** in a **kanban or list view**.
+
+Add property fields
+-------------------
+
+Hover above heading 1 to make the buttons appear. Click on :menuselection:`⚙️Add Properties -->
```
[the remark this answers is not in the document]
Comment (AntoineVDV) [reply content/applications/productivity/knowledge/articles_creation.rst:175]: @jcs-odoo I was thinking about that, and I don't have a good solution for now. So please use emojis for now; just make sure they are compatible with recent browsers.

File: content/applications/productivity/knowledge/properties.rst [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+   * - :guilabel:`Date & Time`
+     - Allows selecting a date and time.
+   * - :guilabel:`Selection`
+     - | Add a drop-down selection menu with restricted values that have been set at the property
+         creation.
+       | To set it up, click :menuselection:`Add a property --> Field Type --> Selection -->
+         Add values --> Enter predetermined values --> Press enter to validate --> repeat --> Click
+         anywhere to close the property creation window`. You can enter as many values as needed.
+   * - :guilabel:`Tags`
+     - | Allows creating as many tags as needed. You can choose from already created tags, or create
```
Comment (chiaraprattico) [inline content/applications/productivity/knowledge/properties.rst:53]: Allows creating as many tags as needed. You can choose from existing tags or create

**Review by chiaraprattico — CHANGES_REQUESTED** [review]

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 6]
```diff
@@ -0,0 +1,265 @@
+================
+Articles editing
+================
+
+Add and style content
+=====================
+
+To start adding content, click anywhere on the page. Your cursor is automatically set to write the
+article's first-level header. Once you are done with the title, press **enter** to move to the next
+line.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst:10]: not here, it's not something shown on-screen but an enter on a keyboard, I could precise it's on their keyboard, though

File: content/applications/productivity/knowledge.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+=========
+Knowledge
+=========
+
+**Odoo Knowledge** is a multipurpose productivity app that allows internal users to enrich their
+business knowledge base and provide individually or collaboratively gathered information.
+
+The pages on which they gather content are called *articles*, and are mainly composed of a title and
+a body. The latter is an HTML field containing text, images, links to other articles, records from
+other models, templates, etc.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge.rst:15]: I changed it a bit, tell me if that's fine by you :)

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+
+  - To choose from a predefined theme color, click :guilabel:`Theme` and select the desired color.
+  - To customize, click :guilabel:`Solid` and define a color using the wheel, by typing its hex
+    code, or its RGBA values.
+  - To use a gradient, click :guilabel:`Gradient`, choose a predefined gradient or click
+    :guilabel:`Custom` to create a personalized gradient.
+
+- To change the text's size, click on the **size number** and select the desired size;
+- Click on the **lists** icons to turn the paragraph into an unordered or an ordered list, or a
+  checklist;
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst:48]: I made this change: 
- Click on the **lists** icons to turn the paragraph into an unordered list, an ordered list, or a
  checklist;
  
  (an unordered is correct according to[ Merriam webster](https://www.merriam-webster.com/dictionary/unordered#dictionary-entry-1))

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+
+- Changing a paragraph into lists (unordered, ordered, checklists).
+- Changing a paragraph into a header (1 → 6), normal text, `code`, or *quotes*.
+
+Add a cover
++++++++++++
+
+You can customize your article by adding a cover pic and an icon. Hover above the
+:abbr:`h1 (First-level header)`, and click on :guilabel:`Add Cover`. A pop-up window opens to set a
+cover picture.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst:66]: You're right, it'd be more correct, I'm turning my "pic(s)" into "picture(s)"

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 10]
```diff
@@ -0,0 +1,265 @@
...
+           - Use
+         * - :guilabel:`Signature`
+           - Insert your signature.
+
+Content from other app
+======================
+
+Knowledge allows to retrieve content views from other applications. To do so, go to the targeted app
+and create the desired view. Then, click :menuselection:`Favorite --> Insert view in article` and
+select an article. The view in inserted at the bottom of the selected article.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst:252]: When you "create a view," such as a graph you like, a kanban view of your prospects, etc., you can insert it in an article by clicking on favorite --> Insert view.
The last sentence tells that when you select the article in which you wish to insert that view, it's added at the end of the article.

I'll try to rephrase it to make it clearer :)

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+
+.. example::
+   To retrieve the view below, we created it by going to :menuselection:`Sales --> Graph icon -->
+   Pie Chart icon` and inserted it by clicking :menuselection:`Favorite --> Insert view in article`
+   and selecting the *Sales Playbook* article.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst:257]: I think it's correct; I'm stating what I did to get the view (=the image), and how I inserted it in the article, all of it in the past tense as that supposedly happened before.
I'll ask Jon what he thinks about the idea, though, to be sure :)

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+
+   .. image:: articles_editing/view-pie.png
+      :align: center
+      :alt: article view from the Sales app
+
+.. important::
+   Users who do **not** have access to the view will **not** be able to access it in Knowledge even
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst:264]: changed it, @jcs-odoo 'll probably ask why we put it in bold, though
I think you're right as it looks better and I believe emphasizing the app we're working on is fine, but let's see what he thinks

File: content/applications/productivity/knowledge/properties.rst [hunk 13]
```diff
@@ -0,0 +1,82 @@
+==========
+Properties
+==========
+
+Properties are fields containing data and that can be added to articles by any user with **write**
+access. These fields are shared between all the child articles and article items that are under the
+same parent.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/properties.rst:7]: Changed it :)

File: content/applications/productivity/knowledge/properties.rst (hunk 5 above)
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/properties.rst:53]: Changed it :)

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 14]
```diff
@@ -0,0 +1,265 @@
...
+           - Use
+         * - :guilabel:`Signature`
+           - Insert your signature.
+
+Content from other app
+======================
+
+Knowledge allows to retrieve content views from other applications. To do so, go to the targeted app
+and create the desired view. Then, click :menuselection:`Favorite --> Insert view in article` and
+select an article. The view in inserted at the bottom of the selected article.
```
Comment (chiaraprattico) [inline content/applications/productivity/knowledge/articles_editing.rst:252]: The view is inserted at the bottom of the selected article.

**Comment by LoredanaLrpz** [comment]
@jcs-odoo can you please take a look at this? I think you forgot about that PR x)

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 15]
```diff
@@ -0,0 +1,265 @@
...
+Knowledge allows to retrieve content views from other applications. To do so, go to the targeted app
+and create the desired view. Then, click :menuselection:`Favorite --> Insert view in article` and
+select an article. The view is inserted at the bottom of the selected article.
+
+.. example::
+   To retrieve the view below, we created it by going to :menuselection:`Sales --> Graph icon -->
+   Pie Chart icon` and inserted it by clicking :menuselection:`Favorite --> Insert view in article`
+   and selecting the *Sales Playbook* article.
+
+   .. image:: articles_editing/view-pie.png
```
Comment (jcs-odoo) [inline content/applications/productivity/knowledge/articles_editing.rst:259]: in this the focus is on the chart. It isn't very obvious that it is inserted inside an article.
I suggest taking a more contextual screenshot. Having a larger image would also reduce the displayed height (here it looks too big)

File: content/applications/productivity/knowledge/properties.rst [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+Properties
+==========
+
+Properties are fields containing data and that can be added to articles by any user with **write**
+access. These fields are shared between all the child articles and article items under the same
+parent.
+
+.. note::
+   To be able to add properties, an article must be either a **child article** or an **article
+   item**.
```
Comment (jcs-odoo) [inline content/applications/productivity/knowledge/properties.rst:11]: I assume you'll add custom links once the pages describing this are created

File: content/applications/productivity/knowledge/properties.rst [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+   * - :guilabel:`Many2one`
+     - | Choose from a list of records that result from a model's domain. You can only select
+         one result.
+       | To set it up, click :menuselection:`Choose a model --> filter the results or show all
+         records --> choose from those results`.
+   * - :guilabel:`Many2many`
+     - | Choose from a list of records that result from a model's domain. You can select as
+         many results as needed.
+       | To set it up, click :menuselection:`Choose a model --> filter the results or show all
+         records --> choose from those results`.
```
Comment (jcs-odoo) [inline content/applications/productivity/knowledge/properties.rst:67]: I suggest making new paragraphs instead.

It's clearer to read, and we usually don't go to the line in English.

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 18]
```diff
@@ -0,0 +1,265 @@
...
+- Click on the **lists** icons to turn the paragraph into an unordered list, an ordered list, or a
+  checklist;
+- Click the **chain** icon to insert or edit an URL link.
+
+.. image:: articles_editing/style-and-colors.png
+   :align: center
+   :alt: Text editor's toolbox
+
+To format a whole paragraph, type `/` anywhere in the text. Doing so opens the
+:doc:`powerbox </developer/reference/frontend/odoo_editor>`, which allows:
```
Comment (jcs-odoo) [inline content/applications/productivity/knowledge/articles_editing.rst:56]: Not sure that linking the dev doc (powerbox) is a good idea here, as the doc isn't going to be helpful to readers.

(also later in this doc)

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Looking good!

Some small changes left and a optional suggestions.

Cheers :)

File: content/applications/productivity/knowledge/properties.rst (hunk 16 above)
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/properties.rst:11]: I confirm

File: content/applications/productivity/knowledge/articles_editing.rst (hunk 15 above)
Comment (LoredanaLrpz) [reply content/applications/productivity/knowledge/articles_editing.rst]: what do you think about this one?

**Comment by LoredanaLrpz** [comment]
@jcs-odoo what do you think?

**Review by jcs-odoo — APPROVED** [review]
Just change the commit title (and PR too).
- 72 characters max.
- remove "create the first knowledge doc". That's what `[ADD] knowledge` stands for already

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 19]
```diff
@@ -0,0 +1,258 @@
...
+-----------
+
+To stylize the text using the text editor, select the text to format.
+
+Then, you can:
+
+- Change the style by clicking :guilabel:`Normal`. Doing so opens a dropdown menu with multiple
+  styles to choose from (:guilabel:`Normal, Code, Header 1 → Header 6, Quote`);
+- Format the text. Click :guilabel:`B` to put it in bold, :guilabel:`I` to put it in italic,
+  :guilabel:`U` to underline, and :guilabel:`S` to strikethrough);
```
Comment (AntoineVDV) [inline content/applications/productivity/knowledge/articles_editing.rst:31]:
```suggestion
  :guilabel:`U` to underline, and :guilabel:`S` to strikethrough;
```

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 20]
```diff
@@ -0,0 +1,258 @@
...
+  icon:
+
+  - To choose from a predefined theme color, click :guilabel:`Theme` and select the desired color.
+  - To customize, click :guilabel:`Solid` and define a color using the wheel, by typing its hex
+    code, or its RGBA values.
+  - To use a gradient, click :guilabel:`Gradient`, choose a predefined gradient or click
+    :guilabel:`Custom` to create a personalized gradient.
+
+- To change the text's size, click the **size number** and select the desired size;
+- click the **lists** icons to turn the paragraph into an unordered list, an ordered list, or a
```
Comment (AntoineVDV) [inline content/applications/productivity/knowledge/articles_editing.rst:42]:
```suggestion
- Click the **lists** icons to turn the paragraph into an unordered list, an ordered list, or a
```

File: content/applications/productivity/knowledge/articles_editing.rst [hunk 21]
```diff
@@ -0,0 +1,258 @@
...
+   - The articles are responsive, and so are the cover pictures. As a result, the images cannot
+     be repositioned manually to fit a particular screen, as they automatically resize
+     depending on the device.
+
+Add an icon
+-----------
+
+To add an icon, hover above the :abbr:`h1 (First-level header)` and click :guilabel:`Add Icon`.
+Doing this sets a random emoji automatically. To change it, click it and select one from the emoji
+window. To remove it, proceed equally and click the red circled :guilabel:`x`.
```
Comment (AntoineVDV) [inline content/applications/productivity/knowledge/articles_editing.rst:87]: ?
```suggestion
window. To remove it, proceed equally and click the red-circled :guilabel:`x`.
```

**Comment by AntoineVDV** [comment]
@robodoo delegate+

**Comment by LoredanaLrpz** [comment]
@robodoo r+
